### ISSUE TITLE
Need better documentation for 2.0 In-Place Upgrade Maintenance Port Dialog 

### ISSUE THREAD

**Issue by odelljl** [Particular/ServiceControl#1167]
I'm testing the upgrade to Service Control 2.0.  

After installing 2.0, I started the in-place upgrade.  I'm immediately presented with this dialog:

![image](https://user-images.githubusercontent.com/1019316/41099268-fb78fb74-6a2b-11e8-9eee-e1de90abb6e0.png)

I'm am not sure how to proceed:

- This "Upgrade Question" is actually a statement. 
- I'm not sure what the prompt is looking for.  Is it an existing port?  Or something new I need to specify?

The documentation glosses over this:

> Clicking the upgrade link will
> 
> Prompt for any additional information that is required such as values for new mandatory settings introduced in the newer version.

I flipped into Maintenance mode (on the non-upgraded version) to see if I was missing something obvious but I did find it.

I am suggesting:

- perhaps rewording the dialog in a future version; and
- updating the upgrade documentation with some more details on this prompt.

**Comment by WilliamBZA** [comment]
Thanks for the feedback @odelljl, I'll tweak the wording in the installer for version 2.0.1 and will also add some more detail to the documentation too.

**Comment by danielmarbach** [comment]
@odelljl what do you think about this proposal

![image](https://user-images.githubusercontent.com/174258/41107823-6354b228-6a73-11e8-9fcb-9403bea5af91.png)


Would that be clearer?

**Comment by odelljl** [comment]
That certainly explains it much better. Is there a default port number that you could provide? In the same way that you default to a high port number for the API connection.

This might be more obvious for people that use Raven a lot. For me it was all new as I don't use it in my regular development.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by danielmarbach** [comment]
@odelljl Thanks for your input. Very valid concerns. We discussed that as well when we implemented the feature and we came to the conclusion that customers might have different philosophies how they want to handle ports. For example, the default port could be +1 to the API part. But maybe customers prefer using a different port range for maintenance operations. So we ended up leaving the choice to the customer. But you bring up good points that indicate maybe we've made the wrong decision and we should provide a default value but then we still would need to ask for input. 

Another option that we discussed was to only ask for it when you put ServiceControl in maintenance mode. We came to the conclusion though that this operation might not have the proper rights to do the URL ACLs so we decided for asking for it upfront. Which is a bit of a pity because you might never need it but we still bother you with that decision. 

Happy to hear your thoughts on that topic and what would have been helpful for you.
